### PR TITLE
Reboot command is working again

### PR DESCRIPTION
**Pull request by lzap**
README has been rewritten, some other improvements like logging to pty1
console, nice banner and minimize improvements.

Releasing new images right now, uploading to my site:

http://lzap.fedorapeople.org/zzz/

### CODE REVIEW

**Comment by lzap** [comment]
asciart

**Comment by lzap** [comment]
Oh damn I was too slow taking the screenshot. Interesting but I see now a bug @GregSutcliffe ^^^ Facter failed to parse ipaddress during this boot. Strange.

**Comment by lzap** [comment]
screenshot

**Comment by lzap** [comment]
Btw worked after reboot, had to be some issue with network, but we start the discover host service after network is up. Strange.

File: extra/ovirt-node-plugin-foreman/ovirt-node-plugin-foreman.spec.in [hunk 1]
```diff
@@ -117,5 +128,9 @@ sed -i '/\[Service\]/a Environment="FACTERLIB=/usr/share/ovirt-node-plugin-forem
 %{_sysconfdir}/ovirt-plugins.d
 
 %changelog
+* Wed Oct 30 2013 Lukas Zapletal <lzap+rpm@redhat.com> 0.2.0-1
```
Comment (GregSutcliffe) [inline extra/ovirt-node-plugin-foreman/ovirt-node-plugin-foreman.spec.in]: Cutnpaste error, should be 18th Dec :)

**Comment by GregSutcliffe** [comment]
@lzap both the macaddress and ipaddress facts depend on "ifconfig -a" so I'm surprised it worked without...

**Comment by lzap** [comment]
Cool I have rebased and fixed the log, also I have added one extra line:

```
systemctl enable NetworkManager-wait-online.service
```

which should make nm to wait until an IP address is discovered. This should prevent the error when IP is not yet ready.

**Comment by GregSutcliffe** [comment]
Merged, thnks @lzap!

**Pull request merged**